src/DisplayBoard.cpp

```diff
@@ -101,20 +116,73 @@ std::string drawColumns(const unsigned int maxElementsInRows,
     for (int i = heightCol; i > 0; --i)
     {
         columns += PADDING + drawPadding(maxElementsInRows);
-        columns += drawEmptyLine(sizeCols);
+        columns += VERTICAL;
+        columns += drawRowOfColumns(i, stringClues);
+        columns += VERTICAL;
         columns += "\n";
     }
+    return columns;
+}
+
+std::vector<std::string> getFormattedColumns(const unsigned int sizeCols,
+                                              const Lines cluesCols)
+{
+    std::vector<std::string> stringLines;
+    if(cluesCols.empty() == false)
+        std::for_each(cluesCols.begin(), cluesCols.end(),
```

Comment:
it's better to use casual range based for loop
`for (const auto & clueCol : cluesCols)`
The same for other places.